src/bootstrap/src/core/builder/mod.rs

```diff
@@ -725,17 +725,17 @@ impl Step for Libdir {
         if !builder.config.dry_run() {
             // Avoid deleting the `rustlib/` directory we just copied (in `impl Step for
             // Sysroot`).
-            if !builder.download_rustc() {
+            /*if !builder.download_rustc() {
```

Comment:
@Kobzol: There's this thing we still need to figure out as discussed on Zulip.
What can be done to keep let's say a stage1 libgccjit.so for m68k at the step that compiles for x86 (to support the case of cross-compiling rustc)?